src/NzbDrone.Api/Frontend/Mappers/StaticResourceMapper.cs

```diff
@@ -27,6 +27,7 @@ public override bool CanHandle(string resourceUrl)
         {
             return resourceUrl.StartsWith("/Content") ||
                    resourceUrl.EndsWith(".js") ||
+                   resourceUrl.EndsWith(".map") ||
```

Comment:
embedding the source map would kind of defeat it's purpose of trying to minimize the amount of data that needs to be sent to the client.